actix-rt/src/lib.rs

```diff
@@ -1,6 +1,5 @@
 //! A runtime implementation that runs everything on the current thread.
-#![deny(rust_2018_idioms, warnings)]
```

Comment:
leave the idioms in, pls

Reply:
ok. would do

Reply:
sorry i meant just idioms, deny(warnings) is bad

Reply:
ok. I didn't think when adding it back. would remove it later